### DISCUSSION TITLE
Renovate does not find Dockerfile in tests folder

### DISCUSSION THREAD

**Discussion by MShekow** [renovatebot/renovate#32748] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

The issue is [reproduced here](https://github.com/MShekow/renovate-dockerfile-issue). The test repo contains two Dockerfiles
- `src/IntegrationAdapters/DF_Adapter/Dockerfile`
- `tests/Unit.Tests/Dockerfile`

For unknown reasons, in the [PR](https://github.com/MShekow/renovate-dockerfile-issue/pull/1/files), Renovate updates only the first file (`src/IntegrationAdapters/DF_Adapter/Dockerfile`).

The Renovate Bot Logs also indicate this:

`DEBUG: Matched 1 file(s) for manager dockerfile: src/IntegrationAdapters/DF_Adapter/Dockerfile`

I'm not super deep into regular expressions, but I don't understand this behavior. Both preconfigured regexes (for the Dockerfile manager) _should_ match the last segment of the path (`/Dockerfile`) in both cases.

**Comment by rarkins** [comment, marked as answer]
This is an example of why we ask for _minimal_ reproductions, including config. If you had reduced your config to `{}`, you should find that the `tests` Dockerfile is processed. The reason is within https://github.com/MShekow/renovate-dockerfile-issue/blob/7fce4c8eee4f4d4a8046c1184b26524f5d972c7d/renovate.json5#L4

**Reply by MShekow** [reply]
OK, thanks for pointing it out. So because I'm using the `config:recommended` preset, which references the `":ignoreModulesAndTests"` preset, the tests folder is excluded. Got it.
